plugins/services/src/js/containers/service-connection/SDKServiceConnectionContainer.js

```diff
@@ -0,0 +1,270 @@
+            className="clickable"
+            copyText={command}
+            onTextCopy={this.handleTextCopy.bind(this, command)}
+            useTooltip={true}
+          >
+            <Icon id="clipboard" size="mini" ref="copyButton" color="grey" />
+          </ClipboardTrigger>
+        </div>
+        {command}
+      </div>
+    );
+  }
+
+  getJSONEndpoint(endpoint) {
+    return (
+      <ConfigurationMapSection>
+        <ConfigurationMapHeading>
+          {endpoint.getEndpointName()}
+        </ConfigurationMapHeading>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>
+            Address
+          </ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            <span>{this.getClipboardTrigger(endpoint.getAddress())}</span>
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>
+            DNS
+          </ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {this.getClipboardTrigger(endpoint.getDns())}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>
+            VIP
+          </ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {this.getClipboardTrigger(endpoint.getVip())}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+      </ConfigurationMapSection>
+    );
+  }
+
+  getFileEndpoint(endpoint) {
+    return (
+      <ConfigurationMapRow>
+        <ConfigurationMapLabel>
+          {endpoint.getEndpointName()}
+        </ConfigurationMapLabel>
+        <ConfigurationMapValue>
+          <a
+            className="active endpoint-download"
+            download={endpoint.getEndpointName()}
+            href={`data:text/plain;content-disposition=attachment;filename=${endpoint.getEndpointName()};charset=utf-8,${encodeURIComponent(endpoint.getEndpointData())}`}
+          >
+            <span>
+              <Icon
+                id="download"
+                className="endpoint-download-icon"
+                size="mini"
+              />
+              {" "}
+              Download
+            </span>
+          </a>
+        </ConfigurationMapValue>
+      </ConfigurationMapRow>
+    );
+  }
+
+  getJSONEndpoints(endpoints) {
+    const jsonEndpoints = endpoints.filter(endpoint => {
+      return endpoint.isJSON();
+    });
+
+    if (jsonEndpoints.length === 0) {
+      return null;
+    }
+
+    return jsonEndpoints.map(endpoint => {
+      return this.getJSONEndpoint(endpoint);
+    });
+  }
+
+  getFileEndpoints(endpoints) {
+    const fileEndpoints = endpoints.filter(endpoint => {
+      return !endpoint.isJSON();
+    });
+
+    if (fileEndpoints.length === 0) {
+      return null;
+    }
+
+    return (
+      <ConfigurationMapSection>
+        <ConfigurationMapHeading>
+          Files
+        </ConfigurationMapHeading>
+        {fileEndpoints.map(endpoint => {
```

Comment:
I'd rather not mixing the logic and "template"

Reply:
Moved to another function